README.md

```diff
@@ -11,3 +11,5 @@ most of which are based on the
 
 These notebooks use the [Julia](https://julialang.org/) package
 [HomalgProject](https://github.com/homalg-project/HomalgProject.jl).
+
+You can execute the notebooks interactively online using [jupyter nbviewer](https://nbviewer.jupyter.org/github/homalg-project/CapHomalgNotebooks/tree/master/).
```

Comment:
Do you mean execute or view? Actually, GitHub already renders the notebooks. But if the rendering fails one can use the link you provide.

Reply:
One can actually execute the notebooks by clicking the option `execute on Binder` in top right corner.

Reply:
Oh, I was in fact unaware of this. Could you please mention the Binder execution button, otherwise, you can simply give two links. One for viewing and one for execution.

Reply:
Done.